snippets/product/type/configured/product.liquid

```diff
@@ -0,0 +1,109 @@
+                    <div class="col-md-12">
+                        <span class="h4">Price & delivery</span>
+                    </div>
+                </div>
+            </div>
+            <div class="panel-body">
+                <div class="price-block">
+
+                    <dl class="dl-horizontal">
+                        <dt class="text-normal overflow-visible">
+                            <span class="product-pull-left">
+                                <span class="text-default-color">Default price</span>
+                            </span>
+                        </dt>
+
+                        <dd>
+                            <span class="product-pull-right text-normal text-default-color">
+                                <span ng-bind="getDefaultPrice()"></span>
+                            </span>
+                        </dd>
+
+                        <dt class="text-normal overflow-visible">
+                            <span class="product-pull-left">
+                                <span class="text-default-color">Custom changes</span>
+                            </span>
+                        </dt>
+
+                        <dd>
+                            <span class="product-pull-right text-normal text-default-color">
+                                <span ng-bind="getCustomChangesTotal()"></span>
+                            </span>
+                        </dd>
+
+                        <br>
+
+                        <dt class="text-normal overflow-visible">
+                            <span class="product-pull-left">
+                                <span class="text-default-color text-bold">Total</span>
+                            </span>
+                        </dt>
+
+                        <dd>
+                            <span class="product-pull-right">
+                                <span ng-bind="getCurrentTotal()"></span>
+                            </span>
+                        </dd>
+                    </dl>
+
+                </div>
+
+                <div class="delivery-cart">
+                    <div ng-controller="purchaseController" ng-init='loadPaymentPlan({{ settings.available_paymentPlans | json }}, "product", "{{ product.id }}")'>
+                        <p ng-show="shipmentType == 'shipping' && shipmentAddress">Expected to arrive: <strong ng-bind="availability['{{ product.id }}'].expectedArrival | date: 'fullDate'"></strong></p>
+                        <p ng-show="shipmentType == 'pickup' && shipmentFulfillmentCenter">Available for pickup: <strong ng-bind="availability['{{ product.id }}'].availableSince | date: 'fullDate'"></strong></p>
+                        <vc-payment-plan available-payment-plans="::availablePaymentPlans"></vc-payment-plan>
+                        <div class="form-group">
+                            {% include 'product/type/configured/quantity' %}
+                        </div>
+                    </div>
+                </div>
```

Comment:
This part looks exactly the same as for other types. May we reuse the code?

Reply:
done